readme.md

```diff
@@ -680,25 +680,27 @@ A function that transform [`Response`](#response) into an array of items. This i
 ###### \_pagination.paginate
 
 Type: `Function`\
-Default: [`Link` header logic](source/index.ts)
+Default: `(response, allItems, currentItems) => getLinkFromHeaders(response)` (find details [here](source/index.ts))
```

Comment:
This change is unnecessary

Reply:
Okay, I restored it but changed the sentence afterwards to make clear how the signature of the function should be. This should make it easier for users without diving into the code first.